main/tests.py

```diff
@@ -1,22 +1,23 @@
 import pytest
+
+from daycare.models import DayCare
 from dogowner.models import DogOwner
 
 
 @pytest.fixture
 def create_dog_owner_user():
-    return DogOwner.create(email='dogowner@address.com',
-                           username='dogOwnerUser01',
-                           password='password123',
-                           dog_name='dog name',
-                           first_name='test',
-                           last_name='user',
-                           phone_number=1234567890,
-                           dog_race='dog race',
-                           dog_picture_url='https://www.google.com/user1.jpg',
-                           dog_age=4,
-                           dog_weight=2,
-                           dog_gender='M'
-                           )
+    return DogOwner.create(email='dogowner@address.com', username='dogOwnerUser01', password='password123',
+                           dog_name='dog name', first_name='test', last_name='user', phone_number=1234567890,
+                           dog_race='dog race', dog_picture_url='https://www.google.com/user1.jpg', dog_age=4,
+                           dog_weight=2, dog_gender='M')
+
+
+@pytest.fixture
+def create_daycare_user():
+    return DayCare.create(email="test@gmail.com", username="testuser01", password="pass",
+                          name="Puppies", description="This is the first daycare test",
+                          price_per_day=10,
+                          capacity=50, area="Merkaz", city="Tel-Aviv", address="The best street 5")
```

Comment:
same, should delete the function : `create_daycare_user`

Reply:
Removed 'create_daycare_user' fixture.